opossum-tool/OpossumUI
# chore: extract components that display attribution info

### Summary of changes

This diff extracts components from the `AttributionColumn` that display package details into a reusable component.

### Context and reason for change

We want to create a popup to show a diff between and attribution/package info and its origin. This popup should reuse the parts from `AttributionColumn` that display package info.

### How can the changes be tested

Run unit and e2e tests (CI).

Open Opossum on this commit and check that nothing changed in the looks and behaviour of audit and attribution view.
